src/cbc_sdk/platform/__init__.py

```diff
@@ -4,7 +4,7 @@
 
 from cbc_sdk.platform.alerts import (Alert, WatchlistAlert, CBAnalyticsAlert, DeviceControlAlert,
                                      ContainerRuntimeAlert, HostBasedFirewallAlert, IntrusionDetectionSystemAlert,
-                                     GroupedAlert)
+                                     GroupedAlert, AlertSearchQuery, GroupedAlertSearchQuery)
```

Comment:
Why are we adding this to the `__init__`? We don't make Query classes in the package level

Reply:
I was trying to access the query classes for my unit tests, to assert the return types on conversion tests. If acceptable i can do it by __module__ and __name__

Reply:
Yeah for unit testing I would use `from cbc_sdk.platform.alerts import AlertSearchQuery, GroupedAlertSearchQuery` instead of adding it to the `__init__`